app/validators/notices/setup/abstraction-alerts/alert-thresholds.validator.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+/**
+ * Validates data submitted for the `/notices/setup/{sessionId}/abstraction-alert/alert-thresholds` page
```

Comment:
all  the paths `/notices/setup/{sessionId}/abstraction-alert/alert-thresholds` 

Are manually added